addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1776,16 +1778,18 @@ const Wysiwyg = Widget.extend({
                     }, 150);
                 },
             },
-            {
+        ];
+        if (!options.isPortalUser) {
```

Comment:
And then
```suggestion
        if (options.isInternalUser) {
```
That would seems more logical and easier to read no?
> "If you have the group `internal` which is allowing attachment operations, add the image cmd"

Instead of
> "If you have the group `portal`, which we somehow know is opposite to the `internal` group which is the one you need to be allowed attachment operations, then we don't show you the command".